styles.css

```diff
@@ -0,0 +1,158 @@
+body {
+    width: 1024px;
+    margin: 0 auto;
+    background-image: url(http://ges-design.ru/library/collage/background/me/background-15-21.jpg);
+    font-family: 'K2D', sans-serif;
+}
+
+header {
+    position: fixed;
+    width: 1024px;
+    z-index: 1;
+}
+
+div.pass {
+    background: linear-gradient(to bottom, rgba(241, 236, 233, 1) 0%, rgba(184, 204, 198, 1) 100%);
+    border-radius: 10px;
+    position: relative;
+    top: 40px;
+    margin-top: 30px;
+    padding: 10px;
+    height: 350px;
+    box-shadow: 4px 4px 3px 0px rgba(0, 0, 0, 0.59);
+}
+
+div.photo {
```

Comment:
`html/css`: You don't have to add `div. ...` before specific class name like `.photo`.